test/e2e/tests/petnames/petnames-signatures.spec.js

```diff
@@ -14,12 +14,37 @@ const {
   clickName,
   rejectRedesignedSignatureOrTransactionRequest,
 } = require('./petnames-helpers');
+const fs = require('fs');
 
 const SIGNATURE_TYPE = {
   TYPED_V3: 'v3',
   TYPED_V4: 'v4',
 };
 
+async function mockLookupSnap(mockServer) {
+  const tgzPath =
+    'test/e2e/mock-response-data/snaps/name-lookup-example-snap-3.1.1.txt';
+  return [
+    await mockServer
+      .forGet(
+        'https://registry.npmjs.org/@metamask/name-lookup-example-snap/-/name-lookup-example-snap-3.1.1.tgz',
```

Comment:
The issue is that this version can change for each new version of `test-snaps`, so maintaining this may be difficult.

Reply:
thanks for your comment, indeed an issue! I'm working on adding a wildcard to catch any verion. And we can add a script for updating the files. This is the same we did with ppom files